### PR TITLE
Revert "Add the parameter `-Paged` to `Get-Help` to support paging (#13374)"

### PR DESCRIPTION
**Pull request by adityapatwardhan**
This reverts commit 485ec44005bdc4b4d4c410e8ae2ca2010b79b0fe.

# PR Summary

We have decided to revert the change to add the `-Paged` parameter to `Get-Help`

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

**Comment by adityapatwardhan** [comment]
Added the MustHave label as committee recommendation is to revert this change.

**Review by daxian-dbw — APPROVED** [review]

File: assets/files.wxs [hunk 1]
```diff
@@ -3105,11 +3105,8 @@
       <Component Id="cmpC0FFB3F4FB30438082D2DC0F4E4BF12D">
         <File Id="fil8FEEEB3B27DF40148EB85CE6AE1A420B" KeyPath="yes" Source="$(var.ProductSourcePath)\Microsoft.DiaSymReader.Native.$(var.FileArchitecture).dll" />
       </Component>
-      <Component Id="cmp5485813EC16244898CB9A944AC2BC5E9">
-        <File Id="fil6EA784763CF94299BC6E75DF068F05A3" KeyPath="yes" Source="$(var.ProductSourcePath)\Microsoft.PowerShell.Pager.dll" />
-      </Component>
-      <Component Id="cmp9AFE828DDCCC4A0E83C4D7476B353023">
-        <File Id="filBE5924015B7147E5BE1D6CDEB8AFB6C3" KeyPath="yes" Source="$(var.ProductSourcePath)\mscordaccore_$(var.FileArchitecture)_$(var.FileArchitecture)_5.0.20.40711.dll" />
```
Comment (daxian-dbw) [inline assets/files.wxs:3112]: The `5.0.20.40711.dll` was an update in the .NET-preview8 PR (see https://github.com/PowerShell/PowerShell/commit/6a37eb9313077f726c4bca7902466a1f5a6a3f44#diff-a4b82ba0eee41f8e27cca51303335950). I think this is the one that should be kept.

Comment (TravisEz13) [reply assets/files.wxs:3112]: CI will regenerate

**Comment by sdwheeler** [comment]
@adityapatwardhan  (and others) When you revert a change like that has a documentation impact, be sure to open a docs issue. I caught this one and backed out the doc change already.

**Comment by adityapatwardhan** [comment]
@sdwheeler Sure, I will keep it in mind.

**Pull request merged**